cypress/e2e/9-tokenActivation.js

```diff
@@ -0,0 +1,61 @@
+import { bigNumberify } from 'ethers/utils';
+
+describe('Token Activation & Deactivation', () => {
+  beforeEach(() => {
+    cy.login();
+    cy.visit(`/colony/${Cypress.config().colony.name}`);
+  });
+
+  it(`User can activate tokens`, () => {
+    // Open Token Activation popover
+    const amountToActivate = 10;
+
+    // Activate tokens
+    cy.tokenActivation(amountToActivate);
+
+    // Check that the active tokens are correct
+    cy.get('@activatedTokens').then(($activatedTokens) => {
+      const activatedTokens = bigNumberify(
+        $activatedTokens.split(' ')[0].split(',').join(''),
+      )
+        .add(amountToActivate)
+        .toString();
+
+      cy.getBySel('activeTokens', { timeout: 60000 }).then(($tokens) => {
+        const tokens = $tokens.text().split(' ')[0].split(',').join('');
+        expect(tokens).to.eq(activatedTokens);
+      });
+    });
+  });
+
+  it(`User can deactivate tokens`, () => {
+    // Open Token Activation popover
+    const amountToDeactivate = 10;
+    cy.getBySel('tokenActivationButton', { timeout: 120000 }).click();
+
+    // Get the number of inactive tokens
+    cy.getBySel('inactiveTokens', { timeout: 60000 })
+      .invoke('text')
+      .as('deactivatedTokens');
+
+    cy.getBySel('deactivateTokensToggle').click();
+
+    cy.getBySel('activateTokensInput').click().type(amountToDeactivate);
+    // eslint-disable-next-line cypress/no-unnecessary-waiting
+    cy.getBySel('tokenActivationConfirm').click().wait(100000);
```

Comment:
Have you got something better to do @ArmandoGraterol? Joking, that was just for my setup :( I will reduce.

Reply:
Hahahahaha now that you mentioned it :thinking: You may be right!

Yeah, I understand, I also, a lot of times, have to increase the wait and timeout time to accommodate my machine. 

In the case of the waits, we might need to find a middle ground between waiting enough for our machines to catch up and also enough to avoid people with faster machines having to wait too long (without them manually changing the amount of course) or the "env/test-suite/whatever automated thing we are going to run the tests" waiting too long.